### ISSUE TITLE
Search in selected folder and its subfolders

### ISSUE THREAD

**Issue by poustak** [pbek/QOwnNotes#757]
####  enhancement                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
It would be fine if I can search in selected folder and its subfolders.

**Comment by pbek** [comment]
Thank you for your suggestion. That would mean that the content of all subfolders will be treated recursively instead of in and of itself.

**Comment by pbek** [comment]
Phew, that was quites some quirking around! Took a lot of time to get it right...

## 17.10.9
- there now is a new setting for note-subfolders to show all notes in a folder 
  including their subfolders in the *Panels settings*
    - if turned on the *All notes* item in the *Note subfolder* panel is 
      hidden because the root folder already shows all notes

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

**Comment by poustak** [comment]
I can see notes in subfolders until I try to find anything (at "Search or create note"). Then I can see notes from current folder only. I'd prefer if search is made for subfolders too.

**Comment by pbek** [comment]
Strange, seems like there is more work to do to make this happen.

**Comment by pbek** [comment]
## 17.10.10
- the setting for note-subfolders to show all notes in a folder including their subfolders now also makes the note-search find items in all folders

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

**Comment by poustak** [comment]
Great. I can confirm, it works.

**Comment by pbek** [comment]
Great, thank you for testing!

**Comment by tragtragtrag** [comment]
I don't want to open a new issue that's why I write here. I often use the option to search in notes in all folders and subfolders. Unfortunatelly, it seems that the shortcut for this option (ALT + F) does not work. I can use (CTRL + F) but then I can only search within the actual folder.

To be able to search within all folders I have two options: to click (in the folder structure) on "all notes" or choose the option in the menu bar (note --> find notes in all tags/subfolders). Only after that files in all folders can be searched.

If using the option (ALT + F) then only the menu will be activated (as is also the regular case in Windows-programs).

Am I doing something wrong? It would be MUCH more comfortable to have a shortcut to search within all folders and subfolder instead of always using the mouse.

I use Qownnotes on Windows 10 as well as the portable version.

**Comment by tragtragtrag** [comment]
I found another solution: I just changed the shortcut in the setting menu (e.g. to CTRL+G). With the new shortcut it works!

**Comment by pbek** [comment]
Great! :grin:
